### ISSUE TITLE
x/build/maintner/maintnerd/maintapi: TRY= comments (for SlowBots) without a preceding blank line are silently ignored

### ISSUE THREAD

**Issue by laboger** [golang/go#38747]
When using trybots with ppc64/ppc64le I found that the initial trybot settings for a CL are not reset if a later TRY line is specified in the same CL. An example is in
 https://go-review.googlesource.com/c/go/+/229761

On the first run I specified all 4 variations that use ppc64 code generations:
TRY=ppc64,ppc64le,linux-ppc64le-osupower9,aix-ppc64

After a later change I only wanted to retest one for ppc64 big endian, and one for little endian, but all 4 were rerun.

Also, it would be nice to have term like 'allppc64' to run all 4 variations of the ppc64 trybots, or at least have something simpler to specify power9 for linux/ppc64le.

**Comment by dmitshur** [comment]
> After a later change I only wanted to retest one for ppc64 big endian, and one for little endian, but all 4 were rerun.

I've looked into what happened in that CL. Your comment fell into a pitfall that isn't documented https://golang.org/wiki/SlowBots#pitfalls.

As currently implemented, the `TRY=` line must be a separate paragraph from the rest of the comment text. A comment like this:

```
Some text here.
TRY=foo
```

Is currently ignored because of [this line](https://github.com/golang/build/blob/916311cec4e1738aae572fe49767cebdc0351144/maintner/maintnerd/maintapi/api.go#L132). It wouldn't be ignored if written as:

```
Some text here.

TRY=foo
```

I've documented the pitfall in commit [`d2a88387`](https://github.com/golang/go/wiki/SlowBots/_compare/a2147d550437456823341f473a6736945cb1c15d...d2a883875c24418ced81f6709ba0af7c8e930b59) as an immediate step to make things better, but the next step should be to improve TRY= parsing so that such comments aren't silently ignored. I'm quite sure that it was an unintentional bug in the original parsing code.

> Also, it would be nice to have term like 'allppc64' to run all 4 variations of the ppc64 trybots, or at least have something simpler to specify power9 for linux/ppc64le.

This is a small change to implement, it just requires updating the [`slowBotAliases`](https://github.com/golang/build/blob/916311cec4e1738aae572fe49767cebdc0351144/dashboard/builders.go#L26) map. Mind filing a separate issue for this, to make tracking work easier?
